src/crypto/SymmetricCipherGcrypt.h

```diff
@@ -49,7 +49,7 @@ class SymmetricCipherGcrypt : public SymmetricCipherBackend
     static int gcryptMode(SymmetricCipher::Mode mode);
     void setErrorString(gcry_error_t err);
 
-    gcry_cipher_hd_t m_ctx;
+    gcry_cipher_hd_t m_ctx{nullptr};
```

Comment:
This is redundant. `m_ctx` is already initialized to `nullptr` within the initialization list of the constructor.

Reply:
fixed